daemons/ipa-slapi-plugins/ipa-extdom-extop/ipa_extdom_common.c

```diff
@@ -130,25 +154,31 @@ int getpwnam_r_wrapper(size_t buf_max, const char *name,
     return ret;
```

Comment:
before returning handle the logging and setting ENOENT in a switch/case statement:
```
done:
    /* set buf/buf_len */
switch(ret) {
case 0:
    if (result == 0) ret = ENOENT;
    break;
case ERANGE:
    LOG("Buffer too small, increase ipaExtdomMaxNssBufSize.\n");
default:
    break;
}
return ret;